editor/groups_editor.cpp

```diff
@@ -668,12 +659,6 @@ void GroupsEditor::_show_group_dialog() {
 }
 
 void GroupsEditor::_bind_methods() {
-
-	ClassDB::bind_method("_add_group", &GroupsEditor::_add_group);
-	ClassDB::bind_method("_remove_group", &GroupsEditor::_remove_group);
-	ClassDB::bind_method("update_tree", &GroupsEditor::update_tree);
```

Comment:
Why were these removed? This is now causing an error when I add/remove an object from a group

```
ERROR: Error calling method from signal 'update_tree': 'GroupsEditor::update_tree': Method not found.
   at: _process_operation_list (core/undo_redo.cpp:296)
```

Error goes away if I add `ClassDB::bind_method("update_tree", &GroupsEditor::update_tree);`

Reply:
I thought I had undone that change after https://github.com/godotengine/godot/pull/36426#issuecomment-592209600 but it seems I missed it.

Reply:
Don't hesitate to make a PR directly for such things, I can easily forget about a comment I read and answered while busy with other things.

Anyway, I remembered now so I'll fix it. There might be more such wrongly removed bindings, so feel free to send PRs if you stumble on them.

Reply:
Fixed by #36758. I re-reviewed the list of removed bindings that do not start with an underscore and it seems good now.

There might still be bindings starting with underscore that were removed even though they'd still be used by MessageQueue or UndoRedo.